src/main/java/net/sf/jabref/logic/layout/LayoutEntry.java

```diff
@@ -585,14 +585,13 @@ private LayoutFormatter getLayoutFormatterByName(String name) throws Exception {
             String className = strings.get(0).trim();
 
             // Check if this is a name formatter defined by this export filter:
-            if (!prefs.getCustomExportNameFormatters().isEmpty()) {
-                String contents = prefs.getCustomExportNameFormatters().get(className);
-                if (contents != null) {
-                    NameFormatter nf = new NameFormatter();
-                    nf.setParameter(contents);
-                    results.add(nf);
-                    continue;
-                }
+
+            String contents = prefs.getCustomExportNameFormatter(className);
+            if (contents != null) {
```

Comment:
Not much. However, the `continue` prohibits the use of `ifPresent` so there will still be a `if (contents.isPresent())`. Still that is probably better.